frontend/packages/dev-console/integration-tests/support/constants/staticText/global-text.ts

```diff
@@ -1,6 +1,7 @@
 export const adminNavigationMenu = {
   serverless: 'Serverless',
   pipelines: 'Pipelines',
+  shipwright: 'Shipwright',
```

Comment:
There is no admin navigation menu "Shipwright", so please remove this here again. It's also not used.

```suggestion
```